derotation/sample_data.py

```diff
@@ -0,0 +1,54 @@
+"""Fetch and load sample datasets for derotation examples.
+
+This module provides functions for fetching sample data used in examples.
+The data are stored in a remote repository and are downloaded to the user's
+local machine the first time they are used.
+"""
+
+from pathlib import Path
+
+import pooch
+
+# URL to the remote data repository on GIN
+DATA_URL = "https://gin.g-node.org/l.porta/derotation_examples/raw/master/"
+
+# Save data in ~/.derotation/data
+DATA_DIR = Path("~", ".derotation", "data").expanduser()
+DATA_DIR.mkdir(parents=True, exist_ok=True)
+
+# File registry without checksums (just for file validation)
+REGISTRY = {
```

Comment:
Why not also add the checksums? Pooch will tell you what these are the first time you fetch the files.